src/config/generic_development_dependency.go

```diff
@@ -22,15 +21,16 @@ func (g *genericDevelopmentDependency) GetContainerName() string {
 
 // GetDockerConfig returns docker configuration and an error if any
 func (g *genericDevelopmentDependency) GetDockerConfig() (types.DockerConfig, error) {
-	renderedVolumes, err := tools.GetRenderedVolumes(g.config.Config.Volumes, g.appDir, g.config.Name)
-	if err != nil {
-		return types.DockerConfig{}, err
+	volumes := []string{}
+	for _, path := range g.config.Config.Persist {
+		name := util.ToSnake(g.config.Name + "_" + path)
```

Comment:
Do you think it's worthwhile to include the application name in here so it's obvious which application the volume belongs to when I run `docker volume ls`?

Reply:
docker-compse will prefix the volume with the compose project name which is the app name in our case